ZenPacks/zenoss/Import4/migration.py

```diff
@@ -92,6 +107,13 @@ def init_command_parser(cls, parser):
         parser.add_argument('--cc-ip', dest='control_center_ip', default=None,
```

Comment:
We happen to not be using it right now, but it we ever want to use ```imp4util.py``` it will be needed.  I think as long as we have ```imp4util``` around we should keep this flag.